src/desktop/Constraint.cpp

```diff
@@ -62,8 +63,18 @@ void CConstraint::onCommit() {
     const auto COMMITTED = m_pConstraint->current.committed;
 
     if (COMMITTED & WLR_POINTER_CONSTRAINT_V1_STATE_CURSOR_HINT) {
-        m_bHintSet      = true;
-        m_vPositionHint = {m_pConstraint->current.cursor_hint.x, m_pConstraint->current.cursor_hint.y};
+        static auto PXWLFORCESCALEZERO = CConfigValue<Hyprlang::INT>("xwayland:force_zero_scaling");
+
+        m_bHintSet = true;
+
+        auto SCALE = 1.f;
+        const auto window = m_pOwner->getWindow();
```

Comment:
style nit: const values in caps + type
`PWINDOW`

Reply:
K, does that also apply to ISXWL and other primitives except pointers?

Reply:
type isn't a hard mandatory but caps is

Reply:
I also changed scale to lower case since it's not a const value